integrated_channels/canvas/exporters/content_metadata.py

```diff
@@ -69,3 +72,9 @@ def transform_self_enrollment(self, content_metadata_item):  # pylint: disable=u
         Whether to allow students to self enroll. Helps students enroll via link or enroll button.
         """
         return True
+
+    def transform_indexed(self, content_metadata_item):  # pylint: disable=unused-argument
```

Comment:
be nice to add a quick test case or addition to existing test case for exporter?